src/add.js

```diff
@@ -1,21 +1,19 @@
-var _add = require('./internal/_add');
 var _curry2 = require('./internal/_curry2');
 
 
 /**
- * Adds two numbers (or strings). Equivalent to `a + b` but curried.
+ * Adds two numbers. Equivalent to `a + b` but curried.
  *
  * @func
  * @memberOf R
  * @category Math
  * @sig Number -> Number -> Number
- * @sig String -> String -> String
- * @param {Number|String} a The first value.
- * @param {Number|String} b The second value.
- * @return {Number|String} The result of `a + b`.
+ * @param {Number} a
+ * @param {Number} b
+ * @return {Number}
```

Comment:
Is this change controversial? I don't think we should document the fact that `R.add` happens to operate on strings: `R.concat` performs that function.